variegated-hal/src/gpio/gpio_binary_heating_element.rs

```diff
@@ -37,7 +37,6 @@ impl<'a, O: OutputPin> GpioBinaryHeatingElement<'a, O> {
     }
 }
 
-#[async_trait::async_trait]
```

Comment:
This doesn't work. The `async_trait` crate is still needed, because the traits are otherwise not dyn-compatible.